annotation/compiler/src/main/java/com/bumptech/glide/annotation/compiler/AppModuleGenerator.java

```diff
@@ -250,13 +250,12 @@ private MethodSpec generateConstructor(ClassName appGlideModule,
         "Discovered AppGlideModule from annotation: " + appGlideModule);
     // Excluded GlideModule classes from the manifest are logged in Glide's singleton.
     for (String glideModule : libraryGlideModuleClassNames) {
-      ClassName moduleClassName = ClassName.bestGuess(glideModule);
```

Comment:
The only use of the `moduleClassName` variable was just to emit a log line, and even with a successful guess the `toString` of a `ClassName` object is the same as the FQCN input (i.e. `glideModule`).